earthpy/spatial.py

```diff
@@ -888,3 +924,119 @@ def draw_legend(im_ax, titles=None, cmap=None, classes=None, bbox=(1.05, 1)):
         borderaxespad=0.0,
         prop={"size": 13},
     )
+
+
+# @deprecate
+def stack_raster_tifs(band_paths, out_path, arr_out=True):
+    """Take a list of raster paths and turn into an output raster stack
+    numpy array. Note that this function depends upon the stack() function.
+
+    TODO: Instead of returning a file path when arr_out=False, consider
+        returning None since the out_path is already
+        an input given by the user. This will make the output type consistent.
+
+    Parameters
+    ----------
+    band_paths : list of file paths
+        A list with paths to the bands you wish to stack. Bands
+        will be stacked in the order given in this list.
+    out_path : string
+        A path with a file name for the output stacked raster
+         tif file.
+    arr_out : boolean
+        A boolean argument to designate what is returned in the stacked
+        raster tif output.
+
+    Returns
+    -------
+    If arr_out keyword is True:
+        tuple: The first value representing the result of src.read() of the
+        stacked array and the second value
+        representing the result of src.profile of the stacked array.
+    If arr_out keyword is False:
+        str : A path with a file name for the output stacked raster tif file.
+
+    
+    """
+
+    # Throw warning and exit
+    raise Warning("stack_raster_tifs is deprecated. Use stack(). Exiting...")
+    sys.exit()
+
+    # Set default import to read
```

Comment:
@joemcglinchy this code seems to be old and not needed. can we remove it? i suspect it's contributing to the code coverage being lower in this pr.

Reply:
what code... all lines of this function below line 966?

Reply:
yes. i guess this raises a larger question of do we KEEP old code that is deprecated and how does code cov handle that? because we don't want old code to impact our coverage stats. I did ping @mbjoseph about this as i've never deprecated a fun before. it just seems like we shouldn't maintain old code. thoughts?

Reply:
removed deprecated code.